src/common/version.ts

```diff
@@ -1,2 +1,2 @@
-export const VERSION = "0.34.2"
+export const VERSION = "0.35.0"
```

Comment:
more of an idea than a suggestion, but maybe it would make sense to pull the version directly from the `package.json` here to reduce the potential for human error when bumping the version 🤔

Reply:
We do that already yeah. See `scripts/gen-version.js`

Reply:
ahh, cool!